truss/templates/server/model_wrapper.py

```diff
@@ -332,7 +336,7 @@ async def _response_generator():
                     while True:
                         chunk = await asyncio.wait_for(
                             response_queue.get(),
-                            timeout=STREAMING_RESPONSE_QUEUE_READ_TIMEOUT_SECS,
+                            timeout=streaming_read_timeout,
```

Comment:
could you add a comment here about what exactly this timeout refers to? (ie: the gap between responses in a stream must be < streaming_read_timeout